src/routes/jira/events/jira-select-version-router.ts

```diff
@@ -1,6 +1,6 @@
 import { Router } from "express";
 import { JiraJwtTokenMiddleware } from "middleware/jira-jwt-middleware";
-import { JiraSelectVersionGet } from "../select/jira-select-version-get";
+import { JiraSelectVersionGet } from "../server/jira-select-version-get";
```

Comment:
Decided it makes more sense to name this dir server so we can save all the GHE server middleware in one spot.